src/App.js

```diff
@@ -184,11 +183,6 @@ export default function App() {
                   <Route path="/contact" component={Contact} />
                   <Route path="/terms" component={Terms} />
                   <Route path="/privacy" component={Privacy} />
-                  {/* Custom URL for sharing the COVID Response Simulator */}
-                  <Redirect
-                    from="/response-simulator"
-                    to="/tools#covid-response-simulator"
-                  />
```

Comment:
Tested the absence of this `Redirect` element. Using `response-simulator` path would just redirect to the home page.